server/db/models/user.js

```diff
@@ -24,10 +27,33 @@ const User = db.define('user', {
       return () => this.getDataValue('salt')
     }
   },
-  googleId: {
-    type: Sequelize.STRING
-  }
-})
+  isAdmin: {
+    type: Boolean,
```

Comment:
must be sequelize.boolean

Reply:
Added 'Sequelize'

Reply:
thank you Mercedes this looks excellent

Reply:
actually it needs to be capitalized

Reply:
Thanks for your feedback. Boolean is not in caps. Plus the seed file is now updated with additional model info.